salt/cloud/clouds/gce.py

```diff
@@ -2080,6 +2080,7 @@ def attach_disk(name=None, kwargs=None, call=None):
     disk_name = kwargs['disk_name']
     mode = kwargs.get('mode', 'READ_WRITE').upper()
     boot = kwargs.get('boot', False)
+    auto_delete = kwargs.get('auto_delete', None)
```

Comment:
The docs say this defaults to `False`, but here (and below) you're setting this to `None`. These should probably be `False` to be more accurate.

Reply:
Updated the code.